interface/src/app/help/help.tpl.html

```diff
@@ -52,7 +52,14 @@ <h3><u>Genes</u></h3>
     Information about the PAO1 genes used to build this server was obtained
     from the following NCBI website:
     <a href="ftp://ftp.ncbi.nih.gov/gene/DATA/GENE_INFO/Archaea_Bacteria/Pseudomonas_aeruginosa_PAO1.gene_info.gz"
-    target="_blank">NCBI</a>.
+       target="_blank">NCBI</a>.
+    <br>
+    Some genes are connected to many genes while others are connected to fewer
+    genes. Right now the genes in the network view are ranked without
+    background correction so that the view prioritizes the genes that are most
+    connected to the query genes. (If the correction were applied, the view
+    would prioritize genes that are more connected to the query genes than to
+    other genes.)
```

Comment:
This is a great explanation. Thank you for nailing it down!

A couple of things will help make this section more readable:

1. This **Genes** section, unlike all others on this page, has three distinct paragraphs rather than just one. They should each be formatted with `<p>...</p>` instead of having `<br>` to allow the browser to show that separation more clearly. In fact, the rest of the page sections should be corrected to use `<p>` tags rather than just bare text after the headers.
2. The explanation in this last part is now clear to me, but the difference between corrected _vs._ uncorrected ranking could be made still more clear with a little emphasis added, I think:

> Some genes are connected to many genes while others are connected to fewer
   genes. Right now, the genes in the network view are ranked <em>without</em>
   background correction so that the view prioritizes the genes that are most
   connected to the query genes. (If the correction were applied, the view
   would prioritize genes that are <em>more connected</em> to the query genes 
   than they are to other genes.)